### ISSUE TITLE
the nested include task fails with ` "'item' is undefined"` error

### ISSUE THREAD

**Issue by mapuri** [ansible/ansible#17144]
##### ISSUE TYPE

- Bug Report
##### COMPONENT NAME

`include` task
##### ANSIBLE VERSION

```
[vagrant@localhost ~]$ ansible --version
ansible 2.1.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```
##### CONFIGURATION

No changes
##### OS / ENVIRONMENT

Centos 7.2
##### SUMMARY

the nested include task fails with `"'item' is undefined"` error when outer include task uses a loop
##### STEPS TO REPRODUCE

1. create a directory structure and place yml files as described in the gist below
2. run the ansible-playbook to hit the error

https://gist.github.com/mapuri/b4b74e81c531eedb834c5ba81cde449f
##### EXPECTED RESULTS

There shall be no error and the nested includes should work. This was working with ansible 2.1.0.0
##### ACTUAL RESULTS

```
[vagrant@localhost ~]$ ansible-playbook -i ./hosts ./tmp/site.yml -kK
SSH password:
SUDO password[defaults to SSH password]:

PLAY [all] *********************************************************************

TASK [setup] *******************************************************************
ok: [localhost]

TASK [include] *****************************************************************
fatal: [localhost]: FAILED! => {"failed": true, "reason": "'item' is undefined"}

NO MORE HOSTS LEFT *************************************************************
        to retry, use: --limit @./tmp/site.retry

PLAY RECAP *********************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=1
```

**Comment by nitzmahone** [comment]
Interesting- works fine on devel and 2.1.0, but fails on 2.1.1 and stable-2.1 (had to add an actual task def to bar.yml) 

Bisect points at https://github.com/ansible/ansible/commits/73a3a5839b7698a6b2d6ea459b0f65234729627b - @jimi-c / @jctanner, ideas?

**Comment by github-abcde** [comment]
When using loop_control to define a loop_var, the error changes accordingly to name the loop_var as undefined instead of item.

**Comment by jimi-c** [comment]
This was a duplicate of #16990, and was resolved in devel and now stable-2.1 via da3fd2d5880c813e412f509fe909c2094eb6b0cb. This will also be included in 2.1.2.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular issue is resolved.

Thank you!

**Comment by mapuri** [comment]
thanks, I will try it out !! :fireworks:
